### ISSUE TITLE
Broken hadoop package - missing GLIBC_2.14

### ISSUE THREAD

**Issue by marcino239** [big-data-europe/docker-hadoop-spark-workbench#20]
`hadoop fs -ls /
`
shows:
`17/03/25 13:49:32 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
`

Looks like problem is related to missing GLIBC library:

```
ldd /opt/hadoop-2.7.1/lib/native/libhadoop.so
/opt/hadoop-2.7.1/lib/native/libhadoop.so: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /opt/hadoop-2.7.1/lib/native/libhadoop.so)

```

**Comment by earthquakesan** [comment]
Hi @marcino239!

Can you please clarify which docker-compose you used (i.e. which version of hadoop docker image).

**Comment by marcino239** [comment]
Hi @earthquakesan , I'm using latest available in git with hash: 52c0bdf

**Comment by earthquakesan** [comment]
@marcino239 Hi!

I've updated Hadoop and Spark, the issue is not there anymore. If you still have problem, feel free to reopen issue.
```
➜  docker-hadoop-spark-workbench git:(master) docker exec -it namenode bash
root@e7026cdd7ab5:/# hadoop
Usage: hadoop [--config confdir] [COMMAND | CLASSNAME]
  CLASSNAME            run the class named CLASSNAME
 or
  where COMMAND is one of:
  fs                   run a generic filesystem user client
  version              print the version
  jar <jar>            run a jar file
                       note: please use "yarn jar" to launch
                             YARN applications, not this command.
  checknative [-a|-h]  check native hadoop and compression libraries availability
  distcp <srcurl> <desturl> copy file or directories recursively
  archive -archiveName NAME -p <parent path> <src>* <dest> create a hadoop archive
  classpath            prints the class path needed to get the
                       Hadoop jar and the required libraries
  credential           interact with credential providers
  daemonlog            get/set the log level for each daemon
  trace                view and modify Hadoop tracing settings

Most commands print help when invoked w/o parameters.
root@e7026cdd7ab5:/# hadoop fs -ls /
Found 1 items
drwxr-xr-x   - root supergroup          0 2017-04-19 11:34 /user
root@e7026cdd7ab5:/# 
```
